File: 3621-minimum-operations-to-make-array-values-equal-to-k/minimum-operations-to-make-array-values-equal-to-k.go
func minOperations(nums []int, k int) int {
    var arr [101]bool;
    result := 0;
    arr[k] = true;

    for _, num := range nums {
        if num < k {
            return -1;
        }

        if !arr[num] {
            result++;
            arr[num] = true;
        }
    }

    return result;
}

File: 3621-minimum-operations-to-make-array-values-equal-to-k/minimum-operations-to-make-array-values-equal-to-k.java
class Solution {
    public int minOperations(int[] nums, int k) {
        boolean[] arr = new boolean[101];
        int result = 0;
        arr[k] = true;

        for (int num : nums) {
            if (num < k) return -1;
            if (!arr[num]) {
                arr[num] = true;
                result++;
            }
        }

        return result;
    }
}